script.js

```diff
@@ -0,0 +1,122 @@
+// Create two variables called width and height.
+// Each of those variables should hold some number.
+// Then, create a new variable that holds the area of a rectangle, calculated based on the width and height variables.
+// Print it to the console.
+
+const width = 9;
+const height = 3;
+
+const rectangleArea = width * height;
+
+console.log(rectangleArea);
+
+// Create two variables called firstNumber and secondNumber.
+// Assign each of them with a different number.
+// Write a program that swaps both values.
+
+let firstNumber = 5;
+let secondNumber = 10;
+
+const temporaryNumber = firstNumber
+
+firstNumber = secondNumber;
+secondNumber = temporaryNumber;
+
+console.log(firstNumber);
+console.log(secondNumber);
+
+// Create variables that hold grades from three different classes:
+//     Physics
+//     Math
+//     English
+// Calculate the average of the grades.
+// If the average equals 4 out of 5 or higher, print "Good job!".
+// Otherwise, print "Try harder!"
+
+let gradePhysic = 4;
+let gradeMath = 4;
+let gradeEnglish = 4;
+
+averageGrades = (gradePhysic + gradeMath + gradeEnglish) / 3;
+if (averageGrades >= 4) {
+    console.log('Good job!');
+} else {
+    console.log('Try harder!');
+}
+
+// Fix the code:
+
+const currentAge = 30;
+const retirementAgeInPoland = 67;
+
+yearsLeftToRetirenment = retirementAgeInPoland - currentAge;
+
+alert(yearsLeftToRetirenment);
+
+// Fix the code (2 mistakes):
+// the expected result of the above code: tax to pay: 1225
+
+const isPayingVAT = true;
+let VATRateInPoland = 0.23;
+const incomeTaxRate = 0.12;
+let monthlyIncome = 3500;
+
+if (isPayingVAT) {
+    taxToPay = monthlyIncome * VATRateInPoland;
+}
+
+taxToPay = taxToPay + monthlyIncome * incomeTaxRate;
+
+alert(`tax to pay: ${taxToPay}`);
+
+// Declare a variable called isSunny and assign a boolean value to it.
+// Create an if-else statement that checks if it's sunny. If it is, print "Don't forget your sunglasses!".
+// If it's not, print "You might need an umbrella."
+
+const isSunny = true;
+
+if (isSunny === true) {
+    console.log("Don't forget your sunglasses!");
+} else {
+    console.log("You might need an umbrella.");
+}
+
+// Create a variable named currentMonth and assign a number representing the current month.
+// Use an if statement to check the month and print a message based on the season (e.g., "It's winter" for December, January, and February).
+
+let currentMonth = 11;
```

Comment:
You are not changing the value of this variable later on. Because of that, you should use `const` instead of `let`